src/commands/realTouch.ts

```diff
@@ -43,8 +43,8 @@ export async function realTouch(
   subject: JQuery,
   options: RealTouchOptions = {}
 ) {
-  const position = options.x && options.y
-    ? { x: options.x, y: options.y }
+  const position = typeof options.x === 'number' || typeof options.y === 'number'
```

Comment:
IMHO `||` makes sense over `&&` here as JSDoc states that:
> X coordinate to click, relative to the Element. Overrides `position`.

https://github.com/dmtrKovalenko/cypress-real-events/blob/09bffe1819719a4b4e28bc5a112ae58fb9f86987/src/commands/realTouch.ts#L16